database/model_config.go

```diff
@@ -15,6 +18,21 @@ type MySQLConfig struct {
 	Connections ConnectionsConfig
 }
 
+func (m *MySQLConfig) MarshalJSON() ([]byte, error) {
+	type Aux struct {
+		Address     string
+		User        string
+		Password    string
+		Connections ConnectionsConfig
+	}
+	return json.Marshal(Aux{
+		Address:     m.Address,
+		User:        m.User,
+		Password:    "a***z",
```

Comment:
I think this needs to be `mask.Password(..)`

Reply:
![](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2FQYRjw6Jz0jyr1AnPW9%2Fgiphy.gif&f=1&nofb=1)